README.md

```diff
@@ -1,2 +1,4 @@
-# td-tools
+<img title="ThingWeb" alt="Thingweb logo" src="https://raw.githubusercontent.com/eclipse-thingweb/website/master/misc/thingweb_logo.svg" width=300px>
+
+## td-tools
```

Comment:
not an issue of this PR but we should think about consistent naming. Here we use the project name, in others we use "Test Things" (without hyphen) or even a totally different name/description "Thing Description Playground"

Reply:
Another PR is addressing this actually (for this repo)